README.md

```diff
@@ -16,6 +16,26 @@ See https://docs.corda.net/getting-set-up.html.
 
 # Usage
 
+## Running tests inside IntelliJ
+	
+We recommend editing your IntelliJ preferences so that you use the gradle runner - this means that the quasar utils 
+plugin will make sure that some flags (like -javaagent - see "Alternatively..." below) are set for you.
+
+To switch to using the gradle runner, Currently in  (but just search for `runner`)
```

Comment:
This line doesn't make sense any more

Reply:
OMG thank god you lot exist - sorry about that!